src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -15,13 +15,17 @@ public class DriverCommandManager {
     private DriverCommand currentCommand = null;
     private Publisher changePublisher = new Publisher();
     private List<CommandVisitor> visitorList= new ArrayList<CommandVisitor>();
+    private final List<DriverCommand> commandHistory = new ArrayList<>();  // Historia poleceń
```

Comment:
Move history to separate class (it can also implement Subscriber for update logic)